universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorage.java

```diff
@@ -232,7 +243,13 @@ public String getSasToken(String fileName) {
       BlobServiceSasSignatureValues signatureValues =
           new BlobServiceSasSignatureValues(
                   OffsetDateTime.now(zoneId).plus(AZURE_SAS_TOKEN_DURATION), blobSasPermission)
-              .setProtocol(SasProtocol.HTTPS_HTTP); // TODO: Get this to work with HTTPS_ONLY
+              .setProtocol(SasProtocol.HTTPS_HTTP);
```

Comment:
question (blocking): is the TODO no longer valid?

Reply:
Yeah, I think you mentioned in another PR that it isn't possible to use HTTPS in dev mode.

Reply:
ah woops I didn't realize this was the AzuriteClient